_opensearch/cluster.md

```diff
@@ -226,6 +230,28 @@ If that is not the case, and `opensearch-d1` and `opensearch-d2` do not have the
 
 Choosing allocation awareness or forced awareness depends on how much space you might need in each zone to balance your primary and replica shards.
 
+### Replica count enforcement
+
+To enforce an even distribution of shards across all zones and avoid hotspots, you can set the `routing.allocation.awareness.balance` attribute to `true`. This setting can be configured in the opensearch.yml file and dynamically updated using the cluster update settings API:
+
+```json
+PUT _cluster/settings
+{
+  "persistent": {
+    "cluster": {
+      "routing.allocation.awareness.balance": "true"
+    }
+  }
+}
+```
+
+The `routing.allocation.awareness.balance` setting is false by default. When it is set to `true`, the total number of shards for the index must be a multiple of the maximum value of all awareness attribute counts. For example, consider a configuration with two awareness attributes set up — zones and rack IDs. Let's say there are two zones and three rack IDs. The maximum of the number of zones and the number of rack IDs is three. Therefore, the number of shards must be a multiple of three. If it is not, OpenSearch throws a validation exception.
```

Comment:
"... the maximum value of all awareness attribute counts." throws me a bit. Is this the same as "...the highest count for any awareness attribute [included in the configuration]."? In other words, since there are three rack IDs and only two zones, the higher value 3 is used to determine the multiple for dictating total number of shards. Did I get that right? Same with "The maximum number of zones and the number of rack IDs is three." To me a "highest count for either number of zones or number of rack IDs..." makes more sense.

Reply:
Yes, it is. Thank you - I will change that.